api/turing/api/ensembling_job_api_it_test.go

```diff
@@ -384,7 +384,7 @@ func TestIntegrationEnsemblingJobController_ListEnsemblingJob(t *testing.T) {
 			),
 			ensemblingJobService: func() service.EnsemblingJobService {
 				svc := &mocks.EnsemblingJobService{}
-				svc.On("List", mock.Anything, mock.Anything).Return(
+				svc.On("List", mock.Anything, mock.Anything, mock.Anything).Return(
```

Comment:
Update the mock here as well?

Reply:
Yeah updated the rest of this file as well. Thanks for the catch.